crm-project/src/main/java/CRM/entity/Item.java

```diff
@@ -27,7 +28,7 @@ public class Item extends SharedContent {
     private Type type;
 
     private String section;
-    private Long assignedToUserId;
+    private Set<User> assignedToUsers;
```

Comment:
You can not simply write this code, you need the hibernate annotations for it...